include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -1431,7 +1431,8 @@ namespace internal
             switch (dim)
               {
                 case 3:
-                  if (symmetric_evaluate && n_q_points_1d > fe_degree)
+                  if (symmetric_evaluate && n_q_points_1d > fe_degree &&
+                      n_q_points_1d < (3 * fe_degree + 3) / 2)
```

Comment:
This is a very minor point.

What about introducing a `static constexpr` member in the surrounding class?
```
template <...>
class ...
{
public:
  static constexpr bool use_collocation =
    n_q_points_1d > fe_degree && n_q_points_1d < (3 * fe_degree + 3) / 2);
};
```
This would make the control flow slightly more readable:
```
if (symmetric_evaluate && use_collocation)
  {
    // ...
  }
```